src/event/events.rs

```diff
@@ -154,6 +151,10 @@ impl Events {
 
     /// Clearing all `Event` values from container explicitly.
     ///
+    /// # Notes
+    ///
+    /// Events are cleared before every `poll`, it not needed to this.
```

Comment:
Looks like there is a typo here. How about: "... every `poll`, but this is not required."